examples/p4est_3d_dgsem/elixir_euler_sedov.jl

```diff
@@ -0,0 +1,102 @@
+
+using Downloads: download
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations3D(1.4)
+
+"""
+    initial_condition_medium_sedov_blast_wave(x, t, equations::CompressibleEulerEquations3D)
+
+The Sedov blast wave setup based on Flash
+- http://flash.uchicago.edu/site/flashcode/user_support/flash_ug_devel/node184.html#SECTION010114000000000000000
+with smaller strength of the initial discontinuity.
+"""
+function initial_condition_medium_sedov_blast_wave(x, t, equations::CompressibleEulerEquations3D)
+  # Set up polar coordinates
+  inicenter = SVector(0.0, 0.0, 0.0)
+  x_norm = x[1] - inicenter[1]
+  y_norm = x[2] - inicenter[2]
+  z_norm = x[3] - inicenter[3]
+  r = sqrt(x_norm^2 + y_norm^2 + z_norm^2)
+
+  # Setup based on http://flash.uchicago.edu/site/flashcode/user_support/flash_ug_devel/node184.html#SECTION010114000000000000000
+  r0 = 0.21875 # = 3.5 * smallest dx (for domain length=4 and max-ref=6)
+  E = 1.0
+  p0_inner = 3 * (equations.gamma - 1) * E / (3 * pi * r0^2)
```

Comment:
```suggestion
  p0_inner = 3 * (equations.gamma - 1) * E / (4 * pi * r0^2)
```

I believe the original FLASH setup for the Sedov problem uses 
```
p0 = 3 * (equations.gamma - 1) * E / ( (ndims + 1) * pi * r0^2)
```
so in 2D everything is fine because you take 3/3 but in 3D we should use 3/4 for consistency to the tree mesh elixir file